libraries/AP_Math/matrix3.cpp

```diff
@@ -228,6 +228,42 @@ T Matrix3<T>::det() const
 }
 
 template <typename T>
+Matrix3<T> Matrix3<T>::inverse(bool& success) const
```

Comment:
But if we are going with the alternative @tridge proposed I think we would
actually not need this. Or am I wrong?
Em 19 de abr de 2016 10:36 AM, "Gustavo José de Sousa" <
notifications@github.com> escreveu:

> In libraries/AP_Math/matrix3.cpp
> <https://github.com/ArduPilot/ardupilot/pull/3926#discussion_r60231299>:
>
> > @@ -228,6 +228,42 @@ T Matrix3<T>::det() const
> >  }
> >
> >  template <typename T>
> > +Matrix3<T> Matrix3<T>::inverse(bool& success) const
>
> It makes sense. Actually, that was one of my alternatives, but I decided
> to try to be consistent with the way transpose methods work. I'll change it
> to the way you suggested.
>
> Then just make sure it still works with *this == other.
>
> Not sure what you mean here.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/ArduPilot/ardupilot/pull/3926/files/75fe9053f3c4a362f6fea4fcc257a020ee6b4afb..25f06c62eaead8b1d5c7c0ebb731ccceada774dc#r60231299>
>